### PR TITLE
[INTERNAL] ComponentAnalyzer / XMLTemplateAnalyzer detection of views

### PR DESCRIPTION
**Pull request by matz3**
Cherry-pick of https://github.com/SAP/ui5-builder/pull/808

### CODE REVIEW

**Review by flovogt — APPROVED** [review]
LGTM

File: lib/lbt/analyzer/XMLTemplateAnalyzer.js [hunk 1]
```diff
@@ -350,6 +353,54 @@ class XMLTemplateAnalyzer {
...
+						default:
+							log.warn(`Unable to analyze sap.ui5/rootView: Unknown type '${viewType}'`);
+						}
+
+						if (viewTypeExtension) {
+							childViewModuleName = fromUI5LegacyName(viewName, viewTypeExtension);
+						}
+					}
+					if (childViewModuleName) {
+						// console.log("child view detected %s", childViewModuleName);
```
Comment (codeworrior) [inline lib/lbt/analyzer/XMLTemplateAnalyzer.js:400]: log verbose, silly or remove?

**Review by codeworrior — APPROVED** [review]
LGTM (but one minor comment that funnily contradicts my own code legacy :-) )

**Review by RandomByte — APPROVED** [review]

File: lib/lbt/analyzer/XMLTemplateAnalyzer.js (hunk 1 above)
Comment (matz3) [reply lib/lbt/analyzer/XMLTemplateAnalyzer.js:400]: I kept it consistent with the existing code. Should be addressed in general. I guess we have several places where console.log is commented out.

**Pull request merged**